api/cli.py

```
@@ -0,0 +1,283 @@
+    )
+
+    if inspect.returncode == 0:
+        if inspect.stdout.strip() == "false":
+            subprocess.run(
+                ["docker", "start", "falkordb-cgraph"],
+                check=True,
+                capture_output=True,
+            )
+            _stderr("Started existing falkordb-cgraph container")
+    else:
+        subprocess.run(
```

Ruff (0.15.6)

[error] `subprocess` call: check for execution of untrusted input

(S603)